skfp/model_selection/utils.py

```diff
@@ -0,0 +1,87 @@
+import warnings
+from collections.abc import Sequence
+from itertools import chain
+from typing import Any, Optional, Union
+
+import numpy as np
+from rdkit.Chem import Mol
+from sklearn.utils import _safe_indexing
+
+
+def ensure_nonempty_list(subset: list) -> None:
+    """
+    Check if the provided subset is empty.
+    """
+    if len(subset) == 0:
+        raise ValueError("One of the subsets is empty.")
+
+
+def fill_missing_sizes(
```

Comment:
Maybe `validate_train_test_sizes`, to keep uniform with train-valid-test variant?